nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml

```diff
@@ -154,5 +154,20 @@
             <artifactId>jettison</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <version>5.0.6.RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-servlet</artifactId>
```

Comment:
Let's move the non-`test` dependencies above so they are all together with the `compile` dependencies and the `test` dependencies are together. Not a technical necessity, but good for logical grouping and identification.

Reply:
Removed one unnecessary dependency and changed the other to scope `test` (as it's only used in the unit test).